kettle/make_json_test.py

```diff
@@ -78,6 +78,11 @@ def expect(path, start, finish, results, **kwargs):
                      {'name': 't2', 'time': 2.0}])
 
     def test_main(self):
+        # these tests look for dissallowed substrings in the json, so we guarantee
+        # a fixed timestamp that will not produce them :shrug:
+        # https://github.com/kubernetes/test-infra/issues/4825
+        time.time = lambda: 1512507930.230854
```

Comment:
I initially just set now() to this, but then it fails, something else somewhere is using time.time, clobbering it inside the unit test seems to be fine. :man_shrugging:

Reply:
should you just fake `time.time()` in the call below? this *might* affect timestamp globally?

Reply:
This affects it globally *in this set of unit tests*, if you don't it will fail because there is other logic using it. This test is quite brittle, however it will pass if you pin the time to a suitable value like I've done here/

Reply:
Other tests should be run in their own interpreter instance.